source/_integrations/tradfri.markdown

```diff
@@ -1,5 +1,6 @@
 ---
 title: IKEA TRÅDFRI
+simplified_title: IKEA Tradfri
```

Comment:
We should not solve that like this. The title can be simplified automatically, we don't need to configure this.

Reply:
Fair enough. Thought there might be and would probably be easier to maintain.